zerver/openapi/zulip.yaml

```diff
@@ -4524,6 +4524,19 @@ paths:
                                             boolean setting controlled this permission; `true` corresponded to `Admins`,
                                             and `false` to `Everyone`.
                                         - $ref: "#/components/schemas/GroupSettingValue"
+                                    can_add_subscribers_group:
+                                      allOf:
+                                        - description: |
+                                            A [group-setting value](/api/group-setting-values) defining the set of
+                                            users who have permission to add subscribers to channels in the organization.
+
+                                            Realm admins have permission to add subscribers to all public channels and private
+                                            channels they are subscribed to; whether or not they are part of this user group.
```

Comment:
I'm not sure we need the realm administrators special behavior here; presumably the only way realm administrators would not have this permission is if this setting was chosen to not include them.

Reply:
Notably previous values of this setting always included organization administrators.

Reply:
Do we want the realm admin special behaviour for the relevant `stream` setting, I clarified this question at https://chat.zulip.org/#narrow/channel/101-design/topic/Can.20subscribe.20other.20users.20on.20user.20profile/near/2039825, so maybe we can continue this discussion there?

Reply:
I don't think we will need it there, either.

Reply:
I think this PR is basically ready to merge except that we need to remove this special `is_admin` behavior in all places, which I think is:
- Server code
- Web code
- API documentation